src/components/sidebar/index.tsx

```diff
@@ -0,0 +1,65 @@
+import { useEffect, useState } from 'react'
+import Logo from '../icons/logo'
+import Link from './link'
+import { Icon } from '@iconify/react'
+
+const LINKS: Record<string, string> = {
```

Comment:
Make sure you do `target=_blank` on these links so it opens a new tab, because right now clicking on socials will leave the site and take you to the social site.

Reply:
👍